Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -563,232 +642,341 @@ def test_get_emails_for_activity___activity_leader(self):
+        
 
+class Test_AllEventsTest(testCase):
 # # # # # # # # # # # #
 # EVENTS & CLAW TESTS #
 # # # # # # # # # # # #
 
 #    Verify that a student can get all their own chapel events
 #    Endpoint -- api/events/chapel
 #    Expected Status Code -- 200 OK
-#    Expected Respones Body -- list of all chapel events attended by the user 
+#    Expected Response Body -- list of all chapel events attended by the user 
     def test_get_all_chapel_events(self):
         self.session = self.createAuthorizedSession(username, password)
         self.url = hostURL + 'api/events/chapel/'
         response = api.get(self.session, self.url)
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
         try:
             response.json()
         except ValueError:
-            pytest.fail('Expected Json response body, got {0}.'.format(response.text))
+            pytest.fail('Expected Json response body, got {0}.'\
+                .format(response.text))
         if not (type(response.json()) is list):
             pytest.fail('Expected list, got {0}.'.format(response.json()))
 
-#    Verify that an user can get all chapel events by username in specific term (ex: FA16)
-#    Endpoint -- api/events/chapel/:username/:term
+#    Verify that an user can get all chapel events in specific term (ex: FA16)
+#    Endpoint -- api/events/chapel/:term
 #    Expected Status Code -- 200 OK
-#    Expected Respones Body -- list of all chapel events attended by the user during term
+#    Expected Response Body -- list of all chapel events attended by the user
+#    during term
     def test_get_all_chapel_events_during_term(self):
         self.session = self.createAuthorizedSession(username, password)
         self.url = hostURL + 'api/events/chapel/' + term_code + '/'
         response = api.get(self.session, self.url)
 
         if not response.status_code == 200:
-            pytest.fail('Expected 200 OK, got {0}.'.format(response.status_code))
+            pytest.fail('Expected 200 OK, got {0}.'\
+                .format(response.status_code))
+        try:
+            response.json()
+        except ValueError:
+            pytest.fail('Expected Json response body, got {0}.'\
+                .format(response.text))
+        if not (type(response.json()) is list):
+            pytest.fail('Expected list, got {0}.'.format(response.json()))
+
+#    Verify that an user can get all chapel events by username in specific term
+#    (ex: FA16)
+#    Precondition -- API Endpoint doesn't work
+#    Endpoint -- api/events/chapel/:username/:term
+#    Expected Status Code -- 200 OK
+#    Expected Response Body -- list of all chapel events attended by the user
+#    during term
+    @pytest.mark.skipif(not unknownPrecondition, reason = "API endpoint" + \
```

Comment:
While this endpoint is listed in the README.md, I don't see it in ApiControllers/EventController.cs.  I think it was removed because we don't want to show anyone else's chapel attendance.

Reply:
I'll update the README and delete the test.